include/stumpless/error.h

```diff
@@ -140,7 +155,8 @@ struct stumpless_error *stumpless_get_error( void );
  */
 enum stumpless_error_id
 stumpless_get_error_id( const struct stumpless_error *err );
-
+const char *
```

Comment:
Document this function (use the formatting of the others in this file as an example) so that it is clear to users how it behaves.

Reply:
Patched now.